src/lint/track_config.nim

```diff
@@ -65,6 +65,8 @@ proc hasValidOnlineEditor(data: JsonNode; path: Path): bool =
     let checks = [
       hasString(d, "indent_style", path, k, allowed = indentStyles),
       hasInteger(d, "indent_size", path, k, allowed = 0..8),
+      hasString(d, "ace_editor_language", path, k),
+      hasString(d, "highlightjs_language", path, k),
```

Comment:
```suggestion
      hasString(d, "highlightjs_language", path, k, checkIsKebab = true),
```
Do we want this?

Reply:
I don't think so, as `c++` e.g. is an allowed value but not a kebab-case string. We could do lowercase, but as we don't yet have a validator for that, I'd prefer to merge it _as is_ right now.